### PR TITLE
Add links to signup for PyLondinium Girls and Trans*Code.

### PR DESCRIPTION
**Pull request by alexchamberlain**
Demo @ https://alexchamberlain.github.io/pylondinium.github.io/talks/transcode.html

### CODE REVIEW

File: schedule.html [hunk 1]
```diff
@@ -94,23 +94,23 @@ <h1 class="cover-heading red-text">Schedule</h1>
             </tr>
             <tr>
                 <th scope="row" class="text-right">11:20</th>
-                <td><a href="talks/talk-2.html"> The Adventures of pip and pacman: A Tale of Package Managers </a><br />
+                <td><a href="https://pylondinium.org/talks/talk-2.html"> The Adventures of pip and pacman: A Tale of Package Managers </a><br />
```
Comment (mariocj89) [inline schedule.html]: Why this change? It will make harder to test locally.

Comment (alexchamberlain) [reply schedule.html]: Nah - this is a generated file - I just fixed the template to include the base URL; for local development, you just need to run `python generate.py --base http://localhost:8000/`
